---

trajectory_msgs/JointTrajectory trajectory
string response_message